CMakeLists.txt

```diff
@@ -98,7 +98,7 @@ endif()
 set( Boost_USE_STATIC_LIBS ON CACHE STRING "ON or OFF" )
 # Most boost deps get implictly picked up via fc, as just about everything links to fc. In addition we pick up
 # the pthread dependency through fc.
-find_package(Boost 1.67 REQUIRED COMPONENTS program_options unit_test_framework)
+find_package(Boost 1.70 REQUIRED COMPONENTS program_options unit_test_framework)
```

Comment:
What about `EosioTester.cmake.in` & `EosioTesterBuild.cmake.in`

Reply:
Yeah, I guess those should be "fixed", though I was probably subconsciously ignoring them because #9042. But I'm not sure how eagerly we can take that.

Reply:
I have no idea. Maybe input from @larryk85 ?